requirements_py/docs/requirements.txt

```diff
@@ -1 +1 @@
-mkdocs==1.2.3
\ No newline at end of file
+mkdocs==1.5.3
```

Comment:
This lib is used in other files too. Please, upgrade those as well:
./requirements_py/docs/requirements.txt
./analyzer/requirements_py/dev/requirements.txt
./web/requirements_py/dev/requirements.txt